### ISSUE TITLE
Text Clarity: on tool pages to help with readability, consider adding light text-shadow to white text on colored backgrounds. 

### ISSUE THREAD

**Issue by jpailthorpe** [mitmedialab/DataBasic#303]
@rahulbot @kanarinka I know this is late to the game, but I do still notice an issue with the white on color being difficult to read/hard on eyes, not just for wtfcsv, but for other tools as well. Consider adding a super light text shadow property to any text on colored backgrounds. Essentially you are adding a blur effect using a black at 30% opacity. Also, no matter how bright your screen is, the shadow will stop the white from being blown out, so it also fixes unknowns

Here are some comparison screens. You might want to also consider removing icon background entirely. I think this creates a better 1:1 feel with aidan's new logo design on the homepage, and keeps the tool focused on content, rather than aesthetic distraction. 

property is 
`text-shadow: 0px 0px 4px rgba(0,0,0,.3) `

(current)
![screen shot 2017-01-24 at 1 50 57 pm](https://cloud.githubusercontent.com/assets/10051544/22263192/939297f4-e241-11e6-9c14-d28edfd781ad.png)

(with shadow) 
![screen shot 2017-01-24 at 1 51 25 pm](https://cloud.githubusercontent.com/assets/10051544/22263159/7cc8be0e-e241-11e6-97eb-62e3ada92531.png)

(with shadow, without icon)
![screen shot 2017-01-24 at 1 52 27 pm](https://cloud.githubusercontent.com/assets/10051544/22263151/7775bda8-e241-11e6-8bcb-b529ef11cfaa.png)

![screen shot 2017-01-24 at 1 50 10 pm](https://cloud.githubusercontent.com/assets/10051544/22263261/d229e558-e241-11e6-8c77-c8b20eff7a8a.png)

![screen shot 2017-01-24 at 1 50 22 pm](https://cloud.githubusercontent.com/assets/10051544/22263262/d22b2170-e241-11e6-91db-85766686a696.png)

<img width="1436" alt="screen shot 2017-01-24 at 2 39 03 pm" src="https://cloud.githubusercontent.com/assets/10051544/22263326/11221226-e242-11e6-9068-21a41d918f45.png">

**Comment by kanarinka** [comment]
Thanks @jpailthorpe for this suggestion - definitely enhances readability and we've incorporated it.
